airflow/airflow.py

```diff
@@ -47,13 +47,18 @@ def get_dag_data(self, instance, tags):
         )
 
     def get_task_data(self, instance, tags):
+        """
+        See https://github.com/apache/airflow/blob/1.8.1/airflow/utils/state.py#L44 for available
```

Comment:
we're on version 1.10: https://github.com/apache/airflow/blob/v1-10-stable/airflow/utils/state.py#L40

Reply:
Thank you, updated